addons/website/views/res_config_settings_views.xml

```diff
@@ -1,5 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+    <record id="res_config_settings_view_form_web_terms" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form.inherit.account.web.terms</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="account.res_config_settings_view_form"/>
```

Comment:
Website does not depend of `account`, this can't be here.
Is this needed when you installed website and account but not the ecommerce?
Would required a `website_account` bridge but it would probably be overkilled? This module seems to have existed years ago, for other reasons tho.

Reply:
no need, I found an alternative way to go about it, Thank you !